src/graphics/gfx_update.cpp

```diff
@@ -862,6 +863,7 @@ void UpdateGraphics(bool skipRepaint)
 //                End With
 //            Next A
 //        Else
+        if(numBackground > 0)
```

Comment:
I remember there being an error here in my branch (either `devel` or `wip-abstract-controls`) too, but I think I fixed it in a different way. We can see when we merge the branches.